utils/sparse.py

```diff
@@ -154,6 +172,29 @@ def reset_best(self, epoch):
 
         return (pruning_start <= epoch <= pruning_end) or epoch == qat_start
 
+    def _mfac_data_loader(self, train_loader, device, multi_scale, imgsz, gs):
+        def dataloader():
+            loader_type = type(train_loader)
+            mfac_dataloader = loader_type(
+                dataset=train_loader.dataset,
+                batch_size=train_loader.batch_size // 2,
```

Comment:
Why do we halve the batch size? Is it some property of MFAC?

Reply:
@dbogunowicz good question. The original MFAC paper recommended using half the batch size of training for MFAC calculations. We've decided not to include this as an exposed control for now